src/components/bulk-add/BulkAddPage.spec.js

```diff
@@ -30,23 +30,26 @@ describe('Bulk Add Page', () => {
     });
 
     describe('User scans an ISBN', () => {
+        let fakeIsbn;
         beforeEach(async () => {
+            fakeIsbn = (Math.random() * 100000000).toString();
+
             fetchMock.mockResponse(JSON.stringify({
-                'ISBN:0201634554': {
+                [`ISBN:${fakeIsbn}`]: {
```

Comment:
Using random data here can be dangerous.  Because we are asserting the value there is the risk that the string concatenation that builds the expected value is simply the same as the implementation. This doesn't test *correctness* but rather *consistency*. The classic example of this is using a random date and formatting it in the test case but confusing 'M' and 'm'. The test passes but the month and minute are switched. Hardcoding a value doesn't have this problem.

Just something to think about.

Reply:
I hadn't thought about this in that way. I have reverted back to using static data for test cases.